### ISSUE TITLE
got same sentence for most of images 

### ISSUE THREAD

**Issue by mathshangw** [keras-team/keras#15644]
i'm trying to implement image captioning and this is my model but got same sentence for most of images after i tried the result model

lr_schedule = keras.optimizers.schedules.ExponentialDecay(
    initial_learning_rate=4e-4,
    decay_steps=50000,
    decay_rate=0.5)            

def define_model(vocab_size,max_length):
    
    inputs1=Input(shape=(number,))

    fe2 = Dense(256, activation='relu')(inputs1)

    inputs2 = Input(shape=(max_length,))

    se1 = Embedding(vocab_size, 256, mask_zero=True)(inputs2)

    se4 = LSTM(512)(se1)

    se2 = Dropout(0.5)(se4)

    decoder1 = Concatenate()([fe2, se2])

    decoder2 = Dense(256, activation='relu')(decoder1)

    outputs = Dense(vocab_size, activation='softmax')(decoder2)
    
    model=Model(inputs=[inputs1,inputs2],outputs=outputs)

    opt =tf.keras.optimizers.Adam(learning_rate=lr_schedule)

    model.compile(loss='categorical_crossentropy',optimizer=opt, metrics=['accuracy'])

i got after 40 epochs this

loss: 3.9597 - accuracy: 0.3031 - val_loss: 3.8455 - val_accuracy: 0.3159
loss: 3.7803 - accuracy: 0.3174 - val_loss: 3.7639 - val_accuracy: 0.3239
loss: 3.6894 - accuracy: 0.3251 - val_loss: 3.7241 - val_accuracy: 0.3282
loss: 3.6356 - accuracy: 0.3300 - val_loss: 3.7012 - val_accuracy: 0.3315
loss: 3.6006 - accuracy: 0.3331 - val_loss: 3.6860 - val_accuracy: 0.3334
loss: 3.5784 - accuracy: 0.3351 - val_loss: 3.6784 - val_accuracy: 0.3344
loss: 3.5631 - accuracy: 0.3365 - val_loss: 3.6727 - val_accuracy: 0.3351
loss: 3.5532 - accuracy: 0.3375 - val_loss: 3.6701 - val_accuracy: 0.3355
loss: 3.5470 - accuracy: 0.3379 - val_loss: 3.6685 - val_accuracy: 0.3357
loss: 3.5423 - accuracy: 0.3386 - val_loss: 3.6673 - val_accuracy: 0.3359
loss: 3.5396 - accuracy: 0.3388 - val_loss: 3.6666 - val_accuracy: 0.3360
loss: 3.5371 - accuracy: 0.3390 - val_loss: 3.6663 - val_accuracy: 0.3360
....
loss: 3.5335 - accuracy: 0.3394 - val_loss: 3.6657 - val_accuracy: 0.3362
Epoch 00024: val_loss did not improve from 3.66567

**Comment by sanatmpa1** [comment]
@mathshangw,

Can you share a complete standalone code or colab gist to reproduce the issue? Thanks!
